client/src/app/__tests__/AppSpec.js

```diff
@@ -2643,6 +2645,12 @@ function createApp(options = {}, mountFn=shallow) {
 
   let app;
 
+  const flags = options.flags || {
+    [ DISABLE_REMOTE_INTERACTION ]: true
```

Comment:
Why do we need this?

Reply:
Without the flag we open privacy preferences in App tests :joy:

Reply:
🐵

Reply:
![image](https://user-images.githubusercontent.com/28307541/83754065-a3eab180-a66b-11ea-856a-68d81273c1dd.png)